lib/yi_appium_caps_util.rb

```diff
@@ -24,6 +23,13 @@ def update (caps_file_name = './appium.txt')
       end
     end
 
+    def create (platform_name)
+      File.open("./appium.txt","w") do |f|
+        f.puts("[caps]\napp = \" \"\nautomationName = \"YouiEngine\"\ndeviceName = \" \"\nplatformName = \"#{platform_name}\"\nyouiEngineAppAddress = \" \"\n")
```

Comment:
So this just creates a dummy file? You still need to update it manually?
Doesn't seem to handle the specific cases for Android, iOS <10 and iOS >10